src/controller/org.controller/index.js

```diff
@@ -728,14 +719,13 @@ router.get('/org/:shortname/id_quota',
   /*
   #swagger.tags = ['Organization']
   #swagger.operationId = 'orgIdQuota'
-  #swagger.summary = "Retrieves the details about the CVE ID quota information of an organization"
+  #swagger.summary = "Retrieves an organization's CVE ID quota information (accessible to all users)"
   #swagger.description = "
         <h2>Access Control</h2>
-        <p>No roles needed to access the endpoint</p>
+        <p>All registered users can access this endpoint</p>
         <h2>Expected Behavior</h2>
-        <p><b>Secretariat:</b> Can see the CVE ID quota information of any Organization</p>
-        <p><b>Admin User:</b> Can only see the CVE ID quota information of the Organization it belongs to</p>
-        <p><b>Regular User:</b> Can only see the CVE ID quota information of the Organization it belongs to</p>""
+        <p><b>Regular and Admin Users:</b> Retrieves CVE ID quota information of user's organization</p>""
```

Comment:
Make this Regular, CNA & Admin Users. Minor - change "of users's org" to "for user's org"